parts/kuberneteskubelet.service

```diff
@@ -10,6 +10,7 @@ ExecStartPre=/bin/bash /opt/azure/containers/kubelet.sh
 ExecStartPre=/bin/mkdir -p /var/lib/kubelet
 ExecStartPre=/bin/mount --bind /var/lib/kubelet /var/lib/kubelet
 ExecStartPre=/bin/mount --make-shared /var/lib/kubelet
+ExecStartPre=-/sbin/ebtables -t nat --list
```

Comment:
This isn't needed if the feature is disabled. The "-" in front ensures any errors will be ignored, so it has no side effects, even if ebtables is not even installed.